easybuild/tools/github.py

```diff
@@ -888,6 +888,8 @@ def new_pr(paths, ecs, title=None, descr=None, commit_msg=None):
                 main_title = ', '.join(names_and_versions[:3] + ['...'])
 
             title = "{%s}[%s] %s" % (class_label, toolchain_label, main_title)
+        elif commit_msg:
```

Comment:
Actually, you may have a good point there, although efficiency isn't a real concern here.

If a custom commit message is specified, that should determine the PR title, rather than an auto-guessed title.

I'll flip them around, thanks for the suggestion!